docs/telemetry.rst

```diff
@@ -30,6 +30,7 @@ You probably want to gain additional insights from a race. Therefore, we have ad
    searchable-snapshots-stats  Searchable Snapshots Stats  Regularly samples searchable snapshots stats
    shard-stats                 Shard Stats                 Regularly samples nodes stats at shard level
    data-stream-stats           Data Streams Stats          Regularly samples data streams stats
+   master-node-stats           Master Node Stats           Regularly determines the current master node name
```

Comment:
I think the intention was to make this an internal telemetry device, so no need to expose any info of command line arguments here. See https://github.com/elastic/rally/blob/4355771a7d9fa027b83331afc2d5e669942b9a0e/esrally/telemetry.py#L1523 for an example.

Reply:
Thanks, fixed